controllers/v1/programs.js

```diff
@@ -3,13 +3,593 @@
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+        "message": "Successfully removed roles in program scope",
+        "status": 200
+      }
+    */
+
+  /**
+   * Remove roles in program scope
+   * @method
+   * @name removeRolesInScope
+   * @param {Object} req - requested data.
+   * @param {String} req.params._id - program id.
+   * @param {Array} req.body.roles - Roles to be added.
+   * @returns {Array} Program scope roles.
+   */
+
+     async removeRolesInScope(req) {
+      return new Promise(async (resolve, reject) => {
+        try {
+  
+          let programUpdated = await programsHelper.removeRolesInScope(
+            req.params._id,
+            req.body.roles
+          );
+      
+          return resolve(programUpdated);
+  
+        } catch (error) {
+          return reject({
+            status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+            message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+            errorObject: error
+          });
+        }
+      });
+    }
+
+
+
+     /**
+    * @api {post} /project/v1/programs/removeEntitiesInScope/:programId remove entities from programs
+    * @apiVersion 1.0.0
+    * @apiName removeEntitiesInScope
+    * @apiGroup Programs
+    * @apiParamExample {json} Request-Body:
+    * {
+        "entities" : ["5f33c3d85f637784791cd830"]
+      }
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /project/v1/programs/removeEntitiesInScope/5ffbf8909259097d48017bbf
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+        "message": "Successfully removed entities in program scope",
+        "status": 200
+      }
+    */
+
+    /**
+   * Remove entities in program scope
+   * @method
+   * @name removeEntitiesInScope
+   * @param {Object} req - requested data.
+   * @param {String} req.params._id - program id.
+   * @param {Array} req.body.entities - Entities to be added.
+   * @returns {Array} Program scope roles.
+   */
+
+  async removeEntitiesInScope(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+
+        let programUpdated = await programsHelper.removeEntitiesInScope(
+          req.params._id,
+          req.body.entities
+        );
+    
+        return resolve(programUpdated);
+
+      } catch (error) {
+        return reject({
+          status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+          message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+          errorObject: error
+        });
+      }
+    });
+  }
+
+  /**
+  * @api {post} /project/v1/programs/list?page=:page&limit=:limit&search=:search
+  * @apiVersion 1.0.0
+  * @apiName 
+  * @apiGroup Programs
+  * @apiParamExample {json} Request-Body:
+  * {
+      "entities" : ["5f33c3d85f637784791cd830"]
```

Comment:
@Prajwal17Tunerlabs why are entities added in the req body? in list below you are not using the req body